scripts/Muon/GUI/Common/results_tab_widget/results_tab_model.py

```diff
@@ -0,0 +1,233 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+from __future__ import (absolute_import, division, unicode_literals)
+
+from mantid.api import AnalysisDataService, WorkspaceFactory
+from mantid.kernel import FloatTimeSeriesProperty
+import numpy as np
+
+from Muon.GUI.Common.observer_pattern import GenericObserver
+
+# Constants
+DEFAULT_TABLE_NAME = 'ResultsTable'
+ALLOWED_NON_TIME_SERIES_LOGS = ("run_number", "run_start", "run_end", "group",
+                                "period", "sample_temp", "sample_magn_field")
+# This is not a particularly robust way of ignoring this as it
+# depends on how Fit chooses to output the name of that value
+RESULTS_TABLE_COLUMNS_NO_ERRS = ['Cost function value']
+
+
+class ResultsTabModel(object):
+    """Captures the data and operations
+    for the results tab"""
+
+    def __init__(self, fitting_context):
```

Comment:
not for this PR, but it might be worth swapping the fit context for the general one in a later PR. What do you think @Mantid-Matthew

Reply:
Can do. I tend to design so that objects have minimal access to state they don't need to know about but I'll let you and Matthew work that out for the future.